auto_round/autoround.py

```diff
@@ -692,15 +692,15 @@ def _parse_format_to_list(self, format: str) -> list:
                     )
 
                 formats[index] = format
-            elif format == "llmcompressor":
```

Comment:
If AutoRound has no kernels, log a warning and don’t list it as supported in the docs.